Packages/com.unity.inputsystem/InputSystem/Controls/TouchControl.cs

```diff
@@ -35,6 +35,11 @@ public class TouchControl : InputControl<TouchState>
         /// <seealso cref="phase"/>
         public TouchPressControl press { get; set; }
 
+        /// <summary>
+        /// Gets the index of the display that was touched.
+        /// </summary>
+        public IntegerControl displayIndex { get; set; }
```

Comment:
Probably good to link Display API from this documentation since an integer provides little guidance on what it represents? E.g. https://docs.unity3d.com/ScriptReference/Display.html

Reply:
It's been a while, so i do not remember what happened if I switched display target of the game view, but I believe that the value is taken from the display dropdown, and so it does reference it when passing to the unmanaged layer.